content/numpy/concepts/math-methods/terms/arcsin/arcsin.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.arcsin()'
+Description: 'Calculate the inverse sine of each element in an array.'
+Subjects: 
+  - 'Computer Science'
+Tags: 
+  - 'Methods'
+  - 'Trigonometry'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **'.arcsin()'** method is used to calculate the inverse sine of the elements in an array. It is commonly employed when solving trigonometry problems where an angle of a right sided triangle is unknow, but two of the sides are known. The result should always be in the range -pi/2 to pi/2 (radians).
+
+Conversion of radians to degrees is done by the formula $r*180/\pi$.
+
+## Syntax
+
+```pseudo
+numpy.arcsin(x, out=None, where=True)
+```
+
+Parameters:
+
+- `x`: array
+  - y-coordinate on the unit circle which makes input in the range from -1 to 1.
+  - If input is outside range, then nan is returned as a result.
+- `out` (Optional):
+  - Array for result. If not provided or None, a freshly allocated array is returned.
+- `where` (Optional): array_like
+  - The condition (array of boolean values) that determines the elements on which the method is to be applied.
+  - Output that corresponds to `False` for a particular element will be returned as is.
+  - If not provided, the arcsin is calculated for all elements in the input array.
+
+Returns: radian: array
```

Comment:
```suggestion
```

Reply:
Really? We don't want to specify what the function returns?

Reply:
We can specify it in the description 😃